eservice/pdo/sservice/scripts/SServiceCLI.py

```diff
@@ -106,14 +107,14 @@ def StartStorageService(config, block_store) :
     reactor.addSystemEventTrigger('before', 'shutdown', thread_pool.stop)
 
     block = Resource()
-    block.putChild(b'get', WSGIResource(reactor, thread_pool, AppWrapperMiddleware(GetBlockApp(block_store))))
-    block.putChild(b'gets', WSGIResource(reactor, thread_pool, AppWrapperMiddleware(GetBlocksApp(block_store))))
-    block.putChild(b'store', WSGIResource(reactor, thread_pool, AppWrapperMiddleware(StoreBlocksApp(block_store))))
-    block.putChild(b'list', WSGIResource(reactor, thread_pool, AppWrapperMiddleware(ListBlocksApp(block_store))))
-    block.putChild(b'check', WSGIResource(reactor, thread_pool, AppWrapperMiddleware(CheckBlocksApp(block_store))))
+    for (wsgi_verb, wsgi_app) in wsgi_block_operation_map.items() :
+        logger.info('add handler for %s', wsgi_verb)
+        verb = wsgi_verb.encode('utf8')
+        app = AppWrapperMiddleware(wsgi_app(config, block_store, service_keys))
```

Comment:
Just a couple of comments here:
* The service keys are provided to each wsgi app, but they are not used -- except to get the verifying key. I can see however that this is a good starting point for a more sophisticated SService.
* perhaps at some point we could push the `service_keys` into `config` to clean up the interface.

Reply:
this is just to make the interface consistent so we can load them cleanly. that is, it makes it easier to add more operations.